### ISSUE TITLE
Address areas missing accessible text alternative

### ISSUE THREAD

**Issue by missmatsuko** [craftcms/cms#6888]
### Description
- The dropdown button next to the save button doesn't have an accessible label (e.g. text content, aria-label).
![Screenshot of save button with dropdown button to its right.](https://user-images.githubusercontent.com/17884198/93944164-f4ece700-fce8-11ea-8c32-975308e10e8c.png)
- Search input (e.g. at top of entries page)
- Entries status

### Additional info

- Craft version: 3.5.11

**Comment by gcamacho079** [comment]
These have been resolved 👍🏼
